Amplify/Categories/DataStore/Subscribe/MutationEvent.swift

```diff
@@ -53,6 +53,21 @@ public struct MutationEvent: Model {
 
     }
 
+    @available(*, deprecated, message: """
+    Init method without ModelSchema is deprecated, use the other init methods.
```

Comment:
Reword to "Initializing from a model without a ModelSchema is deprecated. Use init(model:modelSchema:mutationType:version:graphQLFilterJSON:) instead."